lib/gelf/logger.rb

```diff
@@ -5,7 +5,7 @@ module LoggerCompatibility
     attr_accessor :formatter
 
     # Use it like Logger#add... or better not to use at all.
-    def add(level, message = nil, progname = nil)
+    def add(level, message = nil, progname = nil, &block)
```

Comment:
Although this can be optional, for code clarity at least I added it back.
It has been removed at https://github.com/Graylog2/gelf-rb/commit/1656370e3964280c8baad374fdbbce44a5dbbdaf#diff-9fe274efe6e812365edf617e12452b0dL8